src/api/spec/services/workflows/yaml_to_workflows_service_spec.rb

```diff
@@ -83,5 +83,14 @@
         expect { subject }.to raise_error(Token::Errors::WorkflowsYamlNotParsable)
       end
     end
+
+    context 'with invalid placeholder variables' do
+      let(:workflows_yml_file) { Rails.root.join('spec/support/files/unparsable_workflows_placeholders.yml').expand_path }
+      let(:payload) { github_extractor_payload }
+
+      it 'raises a user-friendly error' do
+        expect { subject }.to raise_error(Token::Errors::WorkflowsYamlNotParsable, 'Unable to parse .obs/workflows.yml: malformed format string - %S')
```

Comment:
This is definitely better now, although I find the error message still a bit cryptic. Especially since `%S` refers to a string in `format(..)`, although the actual error is happening due to `%u`.